sandbox-tools/merge-repos/src/syncReposToStaging.ts

```diff
@@ -42,6 +49,17 @@ interface SyncRepoToStagingOptions extends SwitchBase {
     cloneTo: string;
     originRepo: string;
     originBranch: string;
+
```

Comment:
Added new `test` switch so you don't have to pass quite as many additional command line arguments when testing locally.